test_scripts/upgrade.sh

```diff
@@ -20,104 +27,136 @@ echo "Checking out $START_BRANCH branch"
 git checkout $START_BRANCH
 
 ## Change to molecule directory on pre 7.0 branches
-
-cd roles/confluent.test/
+if [[ $START_BRANCH != 7* ]]
+then
+  cd roles/confluent.test/
+fi
 
 ## Run Molecule Converge on scenario
 echo "Running molecule converge on $SCENARIO_NAME"
 molecule converge -s $SCENARIO_NAME
 
 ## Change to base of cp-ansible
-cd ../..
+if [[ $START_BRANCH != 7* ]]
+then
+  cd ../..
+fi
 
 ## Checkout ending branch
 echo "Checkout $END_BRANCH branch"
 git checkout $END_BRANCH
 
+## With 7.0.x onwards, cp-ansible is packaged as an ansible collection and hence we need to specify collection path to enable running playbooks
+## https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths
+export ANSIBLE_COLLECTIONS_PATH=../../
 
-## With the 6.2.x release, we have removed upgrade playbooks and now handle everything via reconfiguration.  We've added an if statement so that
-## this single script can handle all scenarios.
+## For 6.2.x onwards, this alias function will be used for running the upgrades
+run_playbook_62x+() {
+  if [[ $1 == 6.[2-9].* ]]
```

Comment:
We may want to run from 5.5 to 6/7 branches

Reply:
That's handled. This particular function will actually be resolving based on the end branch.
If end branch is 62x+, this is the function that'll run. Inside this, based on 6xy or 7xy, it'll use either all.yml or confluent.platform.all.